style.css

```diff
@@ -502,3 +502,472 @@ button:disabled {
   background-color: #c1c7d0;
   color: #5e6c84;
 }
+
+.desktop {
+  display: none;
+}
+
+@media only screen and (min-width: 992px) and (max-width: 1100px) {
```

Comment:
- [ ] **Required:** You have specified a media width range for the desktop version of your website and this would not display well for devices that are larger than the specified width. 
Let's only specify the minimum width and leave out the maximum width, hence any screen larger than the specified width would be applied the style rules.

- [ ] **Required:** After doing the above, please ensure the elements are well positioned following your chosen Figma template.

Reply:
Hello, I just reviewed the specifications and found that the range specified was 992px-1100px.